addon/modifiers/did-update.js

```diff
@@ -58,14 +59,30 @@ import { setModifierManager, capabilities } from '@ember/modifier';
 */
 export default setModifierManager(
   () => ({
-    capabilities: capabilities('3.13', { disableAutoTracking: true }),
+    capabilities: capabilities(gte('3.22.0') ? '3.22' : '3.13', {
+      disableAutoTracking: gte('3.22.0') ? false : true,
+    }),
 
     createModifier() {
       return { element: null };
     },
-    installModifier(state, element) {
+
+    installModifier(state, element, args) {
       // save element into state bucket
       state.element = element;
+
+      if (gte('3.22.0')) {
+        // consume all args
+        for (let i = 0; i < args.positional.length; i++) {
+          // "noop" / consume the arg
+          args.positional[i];
+        }
+
+        for (let key of Object.keys(args.named)) {
+          // "noop" / consume the arg
+          args.named[key];
+        }
```

Comment:
Ya, we definitely shouldn't do this. We should either stick with 3.13 capabilities (which does this for us) or fix the tests to actual consume. 

@pzuraq what did the RFC say about rerunning RE: the arguments changes? Does it mention auto tracking (and therefore infer that consumption is required)?

Reply:
> Ya, we definitely shouldn't do this.

I don't think a did-update hook is possible to exist then :-/  (with capabilities 3.22)
it order for the `update` to run, tracked things _must_ be consumed during install.

Reply:
this _very much a wip_, but I'm gonna be doing these sorts of tests for ember-modifier, too: https://github.com/NullVoxPopuli/ember-render-modifiers/pull/1

Reply:
Where do you do that consumption? Did-update will not consume without running, and it won't run without an update. That leaves the only option to consume during install.

> Moved to draft state

I meant the linked PR :/